helpers/helpers.go

```diff
@@ -78,6 +78,12 @@ var ProtoHelpersFuncMap = template.FuncMap{
 	"kebabCase": func(s string) string {
 		return strings.Replace(xstrings.ToSnakeCase(s), "_", "-", -1)
 	},
+	"contains": func(sub, s string) bool {
+		return strings.Contains(s, sub)
+	},
+	"trimstr": func(cutset, s string) string {
```

Comment:
trim is already a builtin function that trim spaces in templates, there is name collision. So yeah not ideal but needed